f8a_tagger/recipes.py

```diff
@@ -219,10 +221,10 @@ def aggregate(input_keywords_file, no_synonyms=None, use_progressbar=False):  #
                     all_keywords[str(keyword)]['synonyms'] = synonyms
 
     # filter out keywords with low occurrence count
-    if defaults.OCCURRENCE_COUNT_FILTER > 1:
+    if occurrence_count_filter > 1:
         result = {}
         for keyword, value in all_keywords.items():
-            if value.get('occurrence_count', 1) > defaults.OCCURRENCE_COUNT_FILTER:
+            if value.get('occurrence_count', 1) > occurrence_count_filter:
```

Comment:
It's default if occurrence count is not present - assuming that if a keyword is registered, it had to be used somewhere - thus 1.